lib/hanami/cli/generators/app/slice_context.rb

```diff
@@ -29,6 +29,18 @@ def underscored_slice_name
             inflector.underscore(slice)
           end
 
+          # @since 2.1.0
+          # @api private
+          def humanized_slice_name
+            inflector.humanize(slice)
+          end
+
+          # @since 2.1.0
+          # @api private
+          def stylesheet_erb_tag
+            %(<%= stylesheet "#{slice}/app" %>)
```

Comment:
Penny for your thoughts: what do you think about seeing a file named "app" inside a _slice_?

(This is absolutely not intended to block this PR from merging, but I figure now is a good time to briefly consider/confirm our naming choices).

Reply:
@timriley I followed the convention of naming "app" those main artifacts. Similar to `slice/admin/layouts/app.html.erb`.

Reply:
This makes sense! Let's stick with it 👍🏼